bin/kyuubi

```diff
@@ -209,3 +209,15 @@ case $1 in
     usage
     ;;
 esac
+
+function echo_logo() {
+echo "  __  __                           __"
+echo " /\ \/\ \                         /\ \      __ "
+echo " \ \ \/'/'  __  __  __  __  __  __\ \ \____/\_\ "
+echo "  \ \ , <  /\ \/\ \/\ \/\ \/\ \/\ \\\\\ \ '__\`\/\ \ "
+echo "   \ \ \\\\\\\`\\\\\ \ \_\ \ \ \_\ \ \ \_\ \\\\\ \ \L\ \ \ \ "
```

Comment:
This line seems like not correct.

Reply:
Because the  double '\' need to transfer meaning.